### ISSUE TITLE
feat: ensemble lofcz/ai-music-detector with existing ML classifier

### ISSUE THREAD

**Issue by borrvick** [borrvick/sync-safe#32]
## Status
Not implemented.

## What it is
[lofcz/ai-music-detector](https://github.com/lofcz/ai-music-detector) is an open-source ML model trained specifically to detect AI-generated music. Ensembling it with the existing custom classifier would improve detection accuracy and reduce false positives.

### Step 1: Evaluate the model
- Clone/install lofcz/ai-music-detector locally
- Run against FakeMusicCaps and SonicsCal test sets
- Compare performance to existing classifier (precision/recall on our data)
- Only proceed if it adds signal (different error profile from our model)

### Step 2: Ensemble architecture
```python
# services/forensics.py
def _ensemble_probability(
    our_prob: float,
    lofcz_prob: float,
    weight_ours: float = 0.6,
    weight_lofcz: float = 0.4,
) -> float:
    """Weighted average ensemble. Weights tuned on calibration set."""
    return our_prob * weight_ours + lofcz_prob * weight_lofcz
```

### Step 3: Calibration
- Run both models on labeled test set (FakeMusicCaps AI + SonicsCal human)
- Optimize weights to maximize AUC on held-out split
- Document calibration results in tools/calibration_results.md

### Settings additions
```python
ENSEMBLE_WEIGHT_OURS: float = 0.6
ENSEMBLE_WEIGHT_LOFCZ: float = 0.4
ENABLE_LOFCZ_ENSEMBLE: bool = True  # feature flag during rollout
```

### Notes
- Only add if model improves AUC by ≥ 5% on our test set — don't add complexity for marginal gain
- lofcz model may have different audio format requirements — check input spec
- Must be wrapped in @spaces.GPU if it uses PyTorch
- License: check lofcz model license before deploying commercially
